examples/examples/proc_macro.rs

```diff
@@ -24,78 +24,20 @@
 // IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 // DEALINGS IN THE SOFTWARE.
 
-use std::net::SocketAddr;
+//! Example of using proc macro to generate working client and server.
 
-use jsonrpsee::core::{async_trait, client::Subscription, Error};
-use jsonrpsee::proc_macros::rpc;
-use jsonrpsee::types::SubscriptionResult;
-use jsonrpsee::ws_client::WsClientBuilder;
-use jsonrpsee::ws_server::{SubscriptionSink, WsServerBuilder, WsServerHandle};
+use jsonrpsee::{core::RpcResult, proc_macros::rpc};
 
-type ExampleHash = [u8; 32];
-type ExampleStorageKey = Vec<u8>;
+#[rpc(client)]
+pub trait Rpc {
```

Comment:
No rush for this PR but would be good to show how the output of this macro is then used?

Reply:
Ops, seems that I was using this example for internal testing of the proc_macro. Have reverted it to origin/master, thanks!